public/js/components/Charts/Charts.js

```diff
@@ -1,90 +1,65 @@
 import React from 'react';
 import {LineChart, Themes} from 'formidable-charts';
 import {Grid, Row, Col} from 'react-flexbox-grid';
+import Chart from './Chart';
+
 import ChartTheme from '../ChartTheme/theme';
 
 const customisedTheme = Object.assign({}, Themes.simple, ChartTheme);
 
-const Charts = ({charts}) => {
+const Charts = ({charts, updating, time}) => {
+    const xLabelByTime = time === '7days' || time === '30days'
```

Comment:
Yes. As per our discussion, I'll swap this out for a date picker and we'll just talk to the api in time periods.